go-controller/pkg/node/gateway_udn.go

```diff
@@ -42,6 +42,87 @@ type UserDefinedNetworkGateway struct {
 	masqCTMark uint
 }
 
+// UTILS Needed for UDN (also leveraged for default netInfo) in openflowmanager
+
+func (c *openflowManager) getDefaultBridgePorts() ([]bridgeUDNConfiguration, string, string) {
+	return c.defaultBridge.getBridgePorts()
+}
+
+func (c *openflowManager) getExGwBridgePorts() ([]bridgeUDNConfiguration, string, string) {
+	return c.externalGatewayBridge.getBridgePorts()
+}
+
+// UTILS Needed for UDN (also leveraged for default netInfo) in bridgeConfiguration
+
+func (b *bridgeConfiguration) getBridgePorts() ([]bridgeUDNConfiguration, string, string) {
+	b.Lock()
+	defer b.Unlock()
+	netConfigs := make([]bridgeUDNConfiguration, len(b.netConfig))
+	for _, netConfig := range b.netConfig {
+		netConfigs = append(netConfigs, *netConfig)
+	}
+	return netConfigs, b.uplinkName, b.ofPortPhys
+}
+
+func (b *bridgeConfiguration) addNetworkBridgeConfig(nInfo util.NetInfo, masqCTMark uint) {
```

Comment:
why modify the networks bridge config then in the func? Wouldnt it be an error if someone tried to add a network with the same name while one already exists?

Reply:
> why modify the networks bridge config then in the func? Wouldnt it be an error if someone tried to add a network with the same name while one already exists?

so that's not how we call it today, we are only calling this from addNetwork in openflow manager which calls addNetworkBridgeConfig so there is never a situation where we even consider updates; so we are not going to overwrite anything; I can change the logic to reflect this @dceara LMK if you agree with this

Reply:
I added a warning if found instead of updating the value... hopefully we don't expect the ofport to change mid way for a given network patch port.

Reply:
As long as we make sure to delete the netConfig for a given network when we fail to provision the node part of things for that network then we should be fine.  But I'm not sure that's the case.